src/bw.ts

```diff
@@ -0,0 +1,1555 @@
+                "-n",
+                "--number",
+                "-s",
+                "--special",
+                "--length",
+                "-h",
+                "--help",
+              ],
+            },
+          ],
+        },
+      ],
+      options: [
+        {
+          name: ["-u", "--uppercase"],
+          description: "Include uppercase characters",
+          exclusiveOn: [
+            "-p",
+            "--passphrase",
+            "--words",
+            "-c",
+            "--capitalize",
+            "--separator",
+            "--includeNumber",
+            "-h",
+            "--help",
+          ],
+        },
+        {
+          name: ["-l", "--lowercase"],
+          description: "Include lowercase characters",
+          exclusiveOn: [
+            "-p",
+            "--passphrase",
+            "--words",
+            "-c",
+            "--capitalize",
+            "--separator",
+            "--includeNumber",
+            "-h",
+            "--help",
+          ],
+        },
+        {
+          name: ["-n", "--number"],
+          description: "Include numeric characters",
+          exclusiveOn: [
+            "-p",
+            "--passphrase",
+            "--words",
+            "-c",
+            "--capitalize",
+            "--separator",
+            "--includeNumber",
+            "-h",
+            "--help",
+          ],
+        },
+        {
+          name: ["-s", "--special"],
+          description: "Include special characters",
+          exclusiveOn: [
+            "-p",
+            "--passphrase",
+            "--words",
+            "-c",
+            "--capitalize",
+            "--separator",
+            "--includeNumber",
+            "-h",
+            "--help",
+          ],
+        },
+        {
+          name: "--length",
+          description: "Define password length",
+          exclusiveOn: [
+            "-p",
+            "--passphrase",
+            "--words",
+            "-c",
+            "--capitalize",
+            "--separator",
+            "--includeNumber",
+            "-h",
+            "--help",
+          ],
+          args: {
+            name: "length",
+            description: "Output length",
+          },
+        },
+        {
+          name: ["-h", "--help"],
+          description: "Display help for generate command",
+          exclusiveOn: [
+            "-p",
+            "--passphrase",
+            "--words",
+            "-c",
+            "--capitalize",
+            "--separator",
+            "--includeNumber",
+            "-u",
+            "--uppercase",
+            "-l",
+            "--lowercase",
+            "-n",
+            "--number",
+            "-s",
+            "--special",
+            "--length",
+          ],
+          priority: 49,
+        },
+      ],
+    },
+    {
+      name: "encode",
+      description: "Base 64 encode stdin",
+      icon: "https://bitwarden.com/images/icon_32x32.png",
```

Comment:
Can you pull this icon URL into a constant? 
```suggestion
      icon: "https://bitwarden.com/images/icon_32x32.png",
```